ext/fs/30_fs.js

```diff
@@ -776,11 +776,15 @@ function checkOpenOptions(options) {
       (val) => val === true,
     ).length === 0
   ) {
-    throw new Error("OpenOptions requires at least one option to be true");
+    throw new Error(
+      "Cannot open file: `options` requires at least one option to be true",
```

Comment:
Should this use the same quoting style as the other changes in this PR?

```suggestion
      "Cannot open file: 'options' requires at least one option to be true",
```

That said I'm not sure about the wording for these messages. The error is thrown before it was attempted to open the file, so the `Cannot open file` bit is confusing to me. It's good that we're hinting as to which fs option was called. Maybe having that in the stack trace is enough?

Reply:
+1 for the quoting styles. I've noticed a real mixed bag, let's go with single quotes like you have here.

As for the error message, the `checkOpenOption` seems to only be used during the `open` and `openSync` functions, so from a callers perspective, the "File could not be opened" because of this error. I'm happy to drop it though it that's confusing, and of course the stack trace will be there too.

Reply:
I've removed the "Cannot open file" prefix and aligned the quoting styles.